File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        res = float('inf')
        for num in nums:
            tot = 0
            while num > 0:
                tot += (num % 10)
                num //= 10
            res = min(res, tot)
        return res

File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        int res = INT_MAX;
        for(int i = 0; i < nums.size(); i++)
        {
            res = min(res, sumDigits(nums[i]));
        }
        return res;
    }
private:
    int sumDigits(int x)
    {
        int res = 0;
        while(x > 0)
        {
            res += (x % 10);
            x /= 10;
        }
        return res;
    }
};